src/content/docs/nrql/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -25,172 +25,37 @@ import queriesNrqlBasicExample from 'images/queries-nrql_screenshot-crop_basic-n
-
-      <td>
-        [Basic and advanced math functions](/docs/insights/nrql-new-relic-query-language/using-nrql/nrql-math-using-select) are supported in the [`SELECT`](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#state-select) statement.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Subqueries
-      </td>
-
-      <td>
-        NRQL supports [subqueries](/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql).
-      </td>
-    </tr>
-    <tr>
-      <td>
-        Comments
-      </td>
-
-      <td>
-        You can add comments to your NRQL queries. [Learn more about comments](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions#comments). 
-      </td>
-    </tr>
-  </tbody>
-</table>
-
-These are the basic rules for NRQL, and we have more information in our [NRQL reference](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/) to help you build your queries.
-
 ## Data exploration [#explore-data]
 
-One of the best ways to learn how to use NRQL is to access a New Relic [query tool](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#where) and use it to explore your data. Here's an example of exploring your data using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) and the suggested entries from the interface. 
+One of the best ways to learn how to use NRQL is to access a New Relic [query tool](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#where) and use it to query your data. Here's an example of exploring your data using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) and the suggested entries from the interface. 
 
 <Callout variant="tip">
-Don't be afraid to play with your data! You won't break anything using any of our query interfaces, so feel free to explore early and often!
+Don't be afraid to play with your data! You won't break anything using any of our query interfaces, so feel free to tinker as much as you like!
 </Callout>
 
-The query begins with `FROM` followed by space. The interface suggests available types of data, and `Transaction` is chosen from that list.
+The query begins with `FROM` followed by a space. The interface suggests available types of data, and you select `Transaction` from that list.
 
 <img
   title="nrql_query01.png"
   alt="A screenshot of a NRQL query beginning with From"
   src={queriesnrqlEventDefinitionsQueryBuilder}
 />
 
-Next, an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) is chosen using `SELECT`, and the query looks as follows:
+Next, you choose an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) using `SELECT`, and the query looks as follows:
```

Comment:
The "you choose" phrasing seems off to me somehow...

Reply:
Agreed, removed the "you" to make it more standard procedural.